pkg/controllers/allocation/filter.go

```diff
@@ -82,3 +84,56 @@ func (f *Filter) matchesProvisioner(pod *v1.Pod, provisioner *v1alpha5.Provision
 	}
 	return fmt.Errorf("matched another provisioner, %s", name)
 }
+
+func (f *Filter) hasSupportedSchedulingConstraints(pod *v1.Pod) error {
```

Comment:
I'm not sure it makes sense to introduce new terminology here? Just wondering if `Supported` might as well be `Valid`? Or is it actually a new concept? Or maybe call this `isValid()`?

Reply:
Yeah I can call this Validate()